### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Tristramg**
Some code chores. It doesn’t change any behaviour

### CODE REVIEW

File: src/validators/check_id.rs [hunk 1]
```diff
@@ -36,25 +36,15 @@ pub fn validate(gtfs: &gtfs_structures::Gtfs) -> Vec<Issue> {
...
-            .map(|agency| make_missing_id_issue(agency))
-            .collect()
-    };
-    r.chain(t)
-        .chain(c)
-        .chain(st)
-        .chain(sh)
-        .chain(ag.into_iter())
-        .collect()
+    let ag = std::iter::empty()
```
Comment (antoine-de) [inline src/validators/check_id.rs]: I don't get why you chain with an empty iterator 🤔

**Review by antoine-de — APPROVED** [review]

File: src/validators/check_id.rs (hunk 1 above)
Comment (Tristramg) [reply src/validators/check_id.rs]: because I break the behaviour! If there is one Agency, we don’t need to check its ID. Thanks for spotting it ;)

Comment (Tristramg) [reply src/validators/check_id.rs]: fixed

**Review by fchabouis — APPROVED** [review]
Merci @Tristramg

**Pull request merged**